fastflix/widgets/panels/status_panel.py

```diff
@@ -111,6 +121,48 @@ def update_bitrate(self, bitrate):
     def update_title_bar(self):
         pass
 
+    def set_started_at(self, msg):
+        try:
+            started_at = datetime.datetime.fromisoformat(msg.split("__")[-1])
+        except Exception as e:
+            logger.warning(f"Unable to parse start time - {str(e)}")
+            return
+
+        self.started_at = started_at
+
+    def update_time_elapsed(self):
+        now = datetime.datetime.now(datetime.timezone.utc)
+
+        if not self.started_at:
+            logger.warning("Unable to update time elapsed because start time isn't set")
+            return
+
+        try:
+            time_elapsed = now - self.started_at
+        except Exception as e:
+            logger.warning(f"Unable to calculate elapsed time - {str(e)}")
+            return
+
+        self.time_elapsed_label.setText(f"{t('Time Elapsed')}: {timedelta_to_str(time_elapsed)}")
+
+    def tick_elapsed_time(self):
+        while self.time_elapsed_ticker_running:
+            self.update_time_elapsed()
```

Comment:
For safety need to change this to a `emit` using a `signal` using a PyQT Thread (like Notifier in `main.py`). With how QT is designed, all widgets should only be updated inside the main PyQT thread loop. (Took me way to long to figure out that was causing some unrelated issues lol) 

This is something I can do post-merge, just let me know if you want to try

Reply:
Yeah I was worried about that actually. Don't have any experience with Qt myself. But I tried to do it the same way it's been done with `LogUpdater` and `Notifier` and pushed it in ccc8e14. Does that work better? 🙂 Anything in particular I should test?